projects/ngx-speculoos/src/lib/route.ts

```diff
@@ -228,6 +233,16 @@ class ActivatedRouteSnapshotStub extends ActivatedRouteSnapshot {
     this._pathFromRoot = value;
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why is this necessary?

Reply:
Because TypeScript complains that I override a property using a getter. But since the property is readonly, and there is no documented constructor to pass the value for this readonly member, I don't really see a betterr choice